packages/kbn-pm/src/utils/bazel/run.ts

```diff
@@ -48,7 +48,11 @@ async function runBazelCommandWithRunner(
   ).subscribe(bazelLogs$);
 
   // Wait for process and logs to finish, unsubscribing in the end
-  await bazelProc;
+  try {
+    await bazelProc;
+  } catch {
+    // no-op
```

Comment:
This is going to prevent the process from exiting with a 1 status code, right? I think we need to throw an instance of `CliError` with a minimal error message, not ignore the error completely.

Reply:
good point! fixed